### PR TITLE
Add OAUTH_SCHEME setting

### PR DESCRIPTION
**Pull request by se-bastiaan**
### Summary

Add OAUTH_SCHEME setting that sets a scheme for OAuth that the app can use.

### How to test
Steps to test the changes you made:
1. Check that you can also add a redirect url with this scheme

### CODE REVIEW

**Comment by se-bastiaan** [comment]
Probably don't need this after all.

**Comment by se-bastiaan** [comment]
Jk, we do.

File: website/thaliawebsite/settings/settings.py [hunk 1]
```diff
@@ -156,8 +156,11 @@
 CORS_URLS_REGEX = r"^/(?:api|user/oauth)/.*"
 
 # OAuth configuration
+OAUTH_SCHEME = os.getenv("OAUTH_SCHEME", "nu.thalia")
```
Comment (pingiun) [inline website/thaliawebsite/settings/settings.py:159]: Maybe rename this to APP_OAUTH_SCHEME to signify it's intended usage

Comment (se-bastiaan) [reply website/thaliawebsite/settings/settings.py]: done

**Review by pingiun — APPROVED** [review]

**Pull request merged**